packages/react-components/src/organisms/TeamCard.tsx

```diff
@@ -63,12 +86,30 @@ const TeamCard: React.FC<TeamCardProps> = ({
   return (
     <EntityCard
```

Comment:
I think that to address Elena's comment on margins (which translates to paddings, in this scenario) you'll need to modify the `EntityCard` to receive the property `overrideStyles`, which in turn would need to be passed down the `Card` component.